tests/Block/AbstractBlockServiceTest.php

```diff
@@ -1,23 +0,0 @@
-<?php
-
-/*
- * This file is part of the Sonata Project package.
- *
- * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-namespace Sonata\BlockBundle\Tests\Block;
-
-use Sonata\BlockBundle\Test\AbstractBlockServiceTestCase;
-
-/**
- * NEXT_MAJOR: remove this class.
- *
- * @deprecated Deprecated since version 3.1. Use Sonata\BlockBundle\Test\AbstractBlockServiceTestCase instead
- */
-abstract class AbstractBlockServiceTest extends AbstractBlockServiceTestCase
-{
-}
```

Comment:
Are you sure this is supposed to be removed?

Reply:
The correct file is on src directory, but I am not sure If I am allowed to remove it from here. 

One thing is for sure: https://github.com/sonata-project/SonataBlockBundle/blob/3.x/composer.json#L47

Reply:
Oh right I didn't notice that.